ingestion/src/metadata/ingestion/ometa/mixins/server_mixin.py

```diff
@@ -67,7 +67,10 @@ def get_server_version(self) -> str:
         Run endpoint /version to check server version
         :return: Server version
         """
-        raw_version = self.client.get("/version")["version"]
+        try:
```

Comment:
when would that happen? Is it when the server is not started?

Reply:
Server had started..
but the version showed up as { }
tried the API got the same response, python was failing with key error..
did the mvn clean package, once that completed, it showed the version.

I started the server from Idea Intellij, that's when one of these edge cases hit.

Reply:
I see, I don't think we should throw a version missmatch error as we don't have any version at all here.

Reply:
fixing it, let me create a VersionNotFound exception

Reply:
Modified